modules/cnv_master/1.0/config/default.yaml

```diff
@@ -0,0 +1,45 @@
+lcr-modules:
+
+    cnv_master:
+        # comma-separated list of modules used to call CNV
+        # this order will also determine the preference which module's output will be used when multiple outputs are available
+        # example: ["battenberg", "sequenza", "cnvkit", "controlfreec"]
+        names: ["__UPDATE__", "__UPDATE__", ..., "__UPDATE__"]
```

Comment:
Why not have these as key:value pairs? e.g. 

```
seg: 
    battenberg: 'results/battenberg-1.2/99-outputs/seg/{seq_type}--projection/{tumour_id}--{normal_id}--{pair_status}.{tool}.{projection}.seg'
    sequenza: 'results/sequenza-1.4/99-outputs/seg/{seq_type}--projection/{tumour_id}--{normal_id}--{pair_status}.{tool}.{projection}.seg'

```
Then you can access the names using `CFG["seg"].keys()'

Reply:
Makes sense, will do!

Reply:
I have switched to the named dictionary here 👍